### ISSUE TITLE
About learning rate setting

### ISSUE THREAD

**Issue by bityangke** [bryanyzhu/two-stream-pytorch#22]
Hi Yi,
How did you decide the lr step ?
Did you follow somewhere else or experiment it youself ?
Thanks in advance!

**Comment by bryanyzhu** [comment]
Hi, I decided the lr step according to this [paper](https://arxiv.org/pdf/1507.02159.pdf). 
But usually in my experiments, I just see when the loss/accuracy saturate, and then decay the lr. I find it more effective.

**Comment by bityangke** [comment]
Thanks very much.

**Comment by bityangke** [comment]
Hi Yi
I noted that Yuanjun used a batchsize（16x4，16 samples on each card） and iter size 4， so the “batch size” is equal to 256. They used step size 4000, 8000, 10000 （iters） ,  the corresponding batch steps should be 1000，2000，2500， and epoch number should be about 27，54， 67.
Am I right？
Thanks！

**Comment by bryanyzhu** [comment]
Hi, I am so sorry for the slow response, I am at CVPR this week.

I am also using similar strategy `iter_size` at [here](https://github.com/bryanyzhu/two-stream-pytorch/blob/master/main_single_gpu.py#L237-L249). So my corresponding batch steps are still 4000, 8000 and 10000, which is about 100, 200 and 250. Hope this is clear.

**Comment by bityangke** [comment]
Thank you！
Hope you have happy CVPR days！
